rhel/SPECS/pmm-update.spec

```diff
@@ -76,6 +76,9 @@ install -p -m 0755 bin/pmm-update %{buildroot}%{_sbindir}/
 # Specifically, the change date is ignored – RPM's "Buildtime" is used instead.
 
 %changelog
+* Wed Sep 08 2021 Nikita Beletskii <nikita.beletskii@percona.com> - 2.22.0-62
```

Comment:
@denisok should it be today or release date?

Reply:
no, date 21st Sep

Reply:
please change

Reply:
no, incorrect. We actually take the date from Repo, so this date doesn't matter.